lib/features/manage_account/data/datasource/identity_data_source.dart

```diff
@@ -13,4 +14,6 @@ abstract class IdentityDataSource {
   Future<bool> deleteIdentity(AccountId accountId, IdentityId identityId);
 
   Future<bool> editIdentity(AccountId accountId, EditIdentityRequest editIdentityRequest);
+
+  Future<Identity> createNewDefaultIdentity(AccountId accountId, CreateNewDefaultIdentityRequest createDefaultIdentityRequest);
```

Comment:
IMO, we don't need to create new function for `default` stuff. How about add a nullable field in `CreateNewIdentityRequest`?